ligo/haskell/test/Test/Ligo/BaseDAO/Proposal.hs

```diff
@@ -160,14 +160,13 @@ test_BaseDAO_Proposal =
           checkTokenBalance unfrozenTokenId dao proposer 48
 
           -- Check total supply
-          let getTotalSupply tokenId consumer = call dao (Call @"Get_total_supply") (mkView tokenId consumer)
-          consumer <- originateSimple "consumer" [] contractConsumer
-          withSender (AddressResolved proposer) $ getTotalSupply unfrozenTokenId consumer
-          checkStorage (AddressResolved $ toAddress consumer) (toVal [148 :: Natural]) -- initial = 200
+          withSender (AddressResolved proposer) $
+            call dao (Call @"Get_total_supply") (mkVoid unfrozenTokenId)
+              & expectError (VoidResult (148 :: Natural)) -- initial = 200
```

Comment:
I wonder why your removed the `-- initial = *` comments from everywhere but here :thinking:

Reply:
Because copypasting is hard! Reverted :)